docs/analysis/influxdb.md

```diff
@@ -0,0 +1,41 @@
+# InfluxDB Metrics
+
+An [InfluxDB](https://www.influxdata.com/) query using [Flux](https://docs.influxdata.com/influxdb/cloud/query-data/get-started/query-influxdb/) can be used to obtain measurements for analysis.
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: AnalysisTemplate
+metadata:
+  name: error-rate
+spec:
+  args:
+  - name: application-name
+  metrics:
+  - name: error-rate
+    # NOTE: To be consistent with the prometheus metrics provider InfluxDB query results are returned as an array.
+    # In the example we're looking at index 0 of the returned array to obtain the value we're using for the success condition
+    successCondition: result[0] <= 0.01
+    provider:
+      newRelic:
```

Comment:
I may just be missing something, but should this be `influxdb`?

Reply:
Oh no, I just copied their docs example